packages/azure-services/src/azure-cosmos/cosmos-client-wrapper.ts

```diff
@@ -157,7 +157,7 @@ export class CosmosClientWrapper {
         try {
             const options: cosmos.RequestOptions = this.getRequestOptionsWithPartitionKey(partitionKey);
             const response = await container.item(id).read(options);
-            const itemT = this.convert<T>(response.body);
+            const itemT = System.convert<T>(response.body);
```

Comment:
>System.convert<T>(response.body); [](start = 26, length = 33)

why is this call needed, when all it does is just cast it to type T?

Reply:
remove activator